requirements.txt

```diff
@@ -4,28 +4,26 @@ SCons==4.5.2
 # NVDA's runtime dependencies
 comtypes==1.2.0
 pyserial==3.5
-wxPython==4.1.1
-git+https://github.com/DiffSK/configobj@3e2f4cc#egg=configobj
+wxPython==4.2.1
+git+https://github.com/DiffSK/configobj@e2ba445#egg=configobj
 requests==2.31.0
-# Required to use a pinned old version for requests.
-# py2exe fails to compile properly without this.
-# This can be removed when upgrading py2exe to 0.13+ and python to 3.8+.
-# https://github.com/Ousret/charset_normalizer/issues/253
-charset-normalizer==2.1.1
+# Pillow is an implicit dependency and requires zlib and jpeg by default, but we don't need it
+Pillow==10.0.1 -C "zlib=disable" -C "jpeg=disable"
 
 #NVDA_DMP requires diff-match-patch
 fast_diff_match_patch==2.0.1
 
 # Packaging NVDA
-py2exe==0.11.1.0
+py2exe==0.13.0.0
+# py2exe fails to compile properly without this.
+# This can be removed when upgrading py2exe to the next release.
+# https://github.com/Ousret/charset_normalizer/issues/253
+# https://github.com/py2exe/py2exe/commit/7b34f06486fbc07500e323796d3f3d5fb862aee3
+charset-normalizer==2.1.1
 
 # For building developer documentation
-sphinx==5.1.1
-sphinx_rtd_theme==1.0.0
-
-# Requirements for type checking.
-# typing_extensions is incorporated in py3.8+, also available via mypy
-typing_extensions==4.3.0

```

Comment:
Can the fact that we no longer bundle `typing_extensions` be mentioned in the Changes for developers section? Some add-ons may import from it.

Reply:
done, also referenced nose being dropped